lib/graphql/execution/execute.rb

```diff
@@ -210,7 +210,7 @@ def resolve_value(owner, parent_type, field_defn, field_type, value, selection,
               )
             when GraphQL::TypeKinds::UNION, GraphQL::TypeKinds::INTERFACE
               query = field_ctx.query
-              resolved_type = query.resolve_type(value)
+              resolved_type = field_type.resolve_type(value, field_ctx)
```

Comment:
:+1: Seems right! Is this change covered by a corresponding change in the tests, to ensure that we _keep_ using the type's `resolve_type` in future versions?